addons/planner_project/models/planner_project.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from openerp import api, models
+
+
+class PlannerProject(models.Model):
+
+    _inherit = 'planner.planner'
+
+    @api.model
+    def _get_planner_application(self):
+        planner = super(PlannerProject, self)._get_planner_application()
+        planner.append(['planner_project', 'Project Planner'])
+        return planner
+
+    @api.model
+    def _prepare_planner_project_data(self):
```

Comment:
This method is not required, since the render method already put `prepare_backend_url`in its default value. If project planner should have a 'prepare' method, this will only add specific value to project planner only.